frontend/src/app/commonComponents/Checkboxes.tsx

```diff
@@ -128,11 +128,12 @@ const CheckboxesFragment = (props: FragmentProps) => {
   const { boxes, name, inputRef, onChange } = props;
 
   return boxes.map(({ value, label, disabled, checked, ...inputProps }) => (
-    <div className="usa-checkbox" key={value}>
+    <div className="usa-checkbox" key={`${name}-${value}`}>
       <input
         className="usa-checkbox__input"
         checked={checked}
-        id={`symptom-${value}`}
+        id={`${name}-${value}`}
```

Comment:
core issue causing the bug was that `symptom-${value}` wasn't a unique ID when there were multiple cards for the same disease. Assuming here that name is a unique identifier so adding it fixed it at least for the test queue.